src/BCEngineMCMC.cxx

```diff
@@ -1631,7 +1606,7 @@ bool BCEngineMCMC::GetNewPointMetropolis()
             #pragma omp parallel for shared(chunk) private(ichain) schedule(static, chunk)
             for (unsigned ichain = 0; ichain < fMCMCNChains; ++ichain) {
                 UpdateChainIndex(ichain);
-                return_value *= GetNewPointMetropolis(ichain, ipar);
+                return_value &= GetNewPointMetropolis(ichain, ipar);
```

Comment:
the bitwise operator `&=` seems appropriate here because it does not short-circuit and we want the rhs to be evaluated.

Reply:
Appropriate or inappropriate? I'm confused by your comment. You are approving of the change or disapproving?